### DISCUSSION TITLE
modelBasePath not able to resolve path at custom library level

### DISCUSSION THREAD

**Discussion by nehasunny123** [vladmandic/human#507] [Q&A]
Hi, I’m developing a custom react library and using this package, 
1. Can you let me know if we can avoid modelBasePath or modelPath and 
2. if I can import json and bin files in my library and then attach them manually to the models object of human instance?
3. Because the problem is when I specify the path it always looks at the parent project level but not the library level ( I have also tried getting the models into the dist folder of my library, and then when I install this library in some other project, it has the models folder under dist in my custom library node modules package but  it doesn’t look at the dist path, it always looks at the parent project where my custom library is installed)

**Comment by vladmandic** [comment]
sorry, not possible at the moment and not planned.
you can always use absolute instead of relative paths.
